tests/test_options_method.py

```diff
@@ -0,0 +1,34 @@
+from flask import Flask, Blueprint
+from flask_jwt_extended import JWTManager, jwt_required, create_access_token
+import pytest
+
+@pytest.fixture(scope='function')
+def app():
+    app = Flask(__name__)
+    app.config['JWT_SECRET_KEY'] = 'secret'
+    JWTManager(app)
+
+    protected_bp = Blueprint('protected', __name__)
```

Comment:
I don't think you need to check this with a blueprint here. Testing it with just an app should be sufficient and a bit cleaner to read.